pallets/pallet-bonded-coins/src/traits.rs

```diff
@@ -82,3 +82,11 @@ where
 		AssetsPallet::<T, I>::set_team(origin.into(), id.into(), issuer.into(), admin.into(), freezer.into())
 	}
 }
+
+/// A trait for getting the next n asset ids to be used during pool creation.
+pub trait NextAssetIds<T: Config> {
+	/// Generic error type.
+	type Error: Into<DispatchError>;
+	/// Get the next `n` asset ids.
+	fn try_get(n: u32) -> Result<BoundedVec<FungiblesAssetIdOf<T>, T::MaxCurrencies>, Self::Error>;
```

Comment:
The `n` parameter already contains the number of IDs to return, so I would have not returned a `BoundedVec` but a simple `Vec`. This is a nit pick but it's also ok the way it is.